### PR TITLE
QPPSF-2883 & QPPSF-2882 & QPPSF-2886 - IA, PI & Quality 2019 measures

### PR DESCRIPTION
**Pull request by shadford**
Added support for 2019 measures generation, and generated PI, IA, Quality Measures for 2019

---

### Summary

Added support for 2019 measures generation, and generated PI, IA, Quality Measures for 2019

### Test Plan

Unit and Integration tests.

### Associated JIRA Tickets

* https://jira.cms.gov/browse/QPPSF-2882
* https://jira.cms.gov/browse/QPPSF-2883
* https://jira.cms.gov/browse/QPPSF-2886

### Reviewers

@NateTheGreatt @bijujoseph @KyleApfel @rockykitamura

### CODE REVIEW

File: scripts/measures/build-measures [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-currentPerformanceYear=2018
+currentPerformanceYear=2019
 # We assume we'll never need to rebuild measures for a previous
 # performance year. If you need to rebuild measures
 # for a previous performance year, you'll have to:
 # 1. use the git history of this file
 # 2. use the git history of any necessary import scripts
 # 3. revert any changes to directory structures below.
 
+if [ $currentPerformanceYear = 2019 ]; then
```
[the remark this answers is not in the document]
Comment (shadford) [reply scripts/measures/build-measures]: Could we wait until the 2018 submission window is complete to clean this up. There's still an outside chance we'll have to modify some of the 2018 measures? i can add a cleanup ticket for 2018 measures for after the submission window is finished

[the remark this answers is not in the document]
Comment (kalvinwang) [reply scripts/measures/build-measures]: @shadford oh, got it, good point. in that case how about let's switch to year-specific build-measures files? like create a new build-measures in /2018 and add an if-statement in this main build-measures for 2018 just like you did for 2019?

[the remark this answers is not in the document]
Comment (shadford) [reply scripts/measures/build-measures]: Ok, I think I have it split apart like we discussed, let me know what you think.

[the remark this answers is not in the document]
Comment (kalvinwang) [reply scripts/measures/build-measures]: I took a closer look and the command we use to run build-measures includes the year, e.g. `npm run build:measures 2018`, so you can simplify lines 3-19 to the following:

```
$currentPerformanceYear = $1
if [ $currentPerformanceYear < 2018 || $currentPerformanceYear > 2019 ]; then
  exec scripts/measures/$currentPerformanceYear/build-measures
  exit
else 
  console.warn('This performance year has no build-measures script: ' + $currentPerformanceYear);
fi
```

Note the comment in lines 4-9 is no longer accurate after this PR

File: scripts/measures/build-2018-measures [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+currentPerformanceYear=$1
```
Comment (kalvinwang) [inline scripts/measures/build-2018-measures]: you can set this to =2018 now, no need to pass it in

also for consistency you can have this file be measures/2018/build-measures

File: scripts/measures/build-2019-measures [hunk 3]
```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-
 currentPerformanceYear=$1
```
Comment (kalvinwang) [inline scripts/measures/build-2019-measures]: you can set this to =2019 now, no need to pass it in

also for consistency you can have this file be measures/2019/build-measures

**Pull request merged**